src/StoreKeeper/WooCommerce/B2C/Backoffice/Pages/Tabs/TaskLogsTab.php

```diff
@@ -259,6 +259,23 @@ public function renderTaskStatus($value, $task)
                 if (isset($errorOutput['exception-reference'])) {
                     echo __('Error reference', I18N::DOMAIN).': '.esc_html($errorOutput['exception-reference']).':<br>';
                 }
+
+                if (isset($errorOutput['exception-difference'])) {
+                    echo ''.__('Extra metadata', I18N::DOMAIN).':<br>';
+                    $extrasDifferences = $errorOutput['exception-difference'];
+                    $shopExtras = $extrasDifferences['shop-extras'];
+                    $backofficeExtras = $extrasDifferences['backoffice-extras'];
+                    if (count($shopExtras) > 0) {
+                        echo '<strong>'.__('Shop', I18N::DOMAIN).'</strong><br>';
+                        $this->generateExtrasTable($shopExtras);
+                    }
+                    if (count($backofficeExtras) > 0) {
+                        echo '<br><strong>'.__('Backoffice', I18N::DOMAIN).'</strong><br>';
+                        $this->generateExtrasTable($backofficeExtras);
+                    }
```

Comment:
```suggestion
                    if (count($shopExtras) > 0 || count($backofficeExtras) > 0) {
                        echo '<strong>'.__('Shop', I18N::DOMAIN).'</strong><br>';
                        $this->generateExtrasTable($shopExtras);
                        echo '<br><strong>'.__('Backoffice', I18N::DOMAIN).'</strong><br>';
                        $this->generateExtrasTable($backofficeExtras);
                    }
```

good to also show if one is empty

Reply:
Changes implemented.